src/Components/Notifications/Notification.tsx

```diff
@@ -38,3 +56,60 @@ export const NotificationFragmentContainer = createFragmentContainer(
     `,
   }
 )
+
+const QUERY = graphql`
+  query NotificationQuery($id: String!) {
+    me {
+      ...Notification_me @arguments(notificationId: $id)
+    }
+  }
+`
+
+export const NotificationQueryRenderer: React.FC = () => {
+  const { state } = useNotificationsContext()
```

Comment:
I store the ID of the currently selected activity/notification in a context. This allows setting the first activity/notification in the list as active from `NotificationsList` for the route `/notifications` (which is a requirement).